src/cljs/athens/keybindings.cljs

```diff
@@ -101,10 +104,20 @@
       (= type :slash) (cond
                         (= :up direction) (do
                                             (.. e preventDefault)
-                                            (swap! state update :search/index dec))
+                                            (swap! state update :search/index #(dec (if (zero? %) (count slash-options) %)))
+                                            (let [cur-index (:search/index @state)
+                                                  container-el (. js/document getElementById "slash-menu-container")
```

Comment:
Would this work, though? I don't see if the actual div has the ID yet.

Reply:
![slash-menu-container](https://user-images.githubusercontent.com/1746118/88605822-1020da80-d049-11ea-9347-b0f358a7a326.png)
I see the id on the buttons' direct parent. If you checked out the code, did it not work for you?

Reply:
I dont' see the id in the code. What is your github pages? Don't see anything on http://alaq.github.io/athens/ 

https://github.com/alaq/athens/blob/slash-scrolling/src/cljs/athens/views/blocks.cljs#L326-L335

Reply:
https://github.com/alaq/athens/blob/d240f3f28e6ad7cbb4212086b009680ccbc5dafc/src/cljs/athens/views/blocks.cljs#L326-L335

Reply:
So sorry, I had other unrelated changes in `blocks.cljs` so I forgot to stage the id...
Let me look at what's wrong with the GitHub pages.